### PR TITLE
fix(components): add expand collapse all to proposal message accordion

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            </Flex>
+          </AccordionButton>
+          <AccordionPanel p={0} pt={4}>
+            <div
+              style={
+                isExpanded
+                  ? { display: "block" }
+                  : { height: 0, display: "none" }
+              }
+            >
```
Comment (songwongtp) [inline src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx:62]: do we need this `div`?

File: src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+interface ProposalMessageCardProps {
+  header: string;
+  jsonString: string;
+}
+
+export const ProposalMessageCard = ({
+  header,
+  jsonString,
+}: ProposalMessageCardProps) => {
+  return (
```
Comment (songwongtp) [inline src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx:22]: remove `return`

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx (hunk 1 above)
Comment (songwongtp) [reply src/lib/pages/proposal-details/components/proposal-overview/ProposalMessageCard.tsx:62]: likely the cause of the snap collapse animation

File: CHANGELOG.md [hunk 3]
```diff
@@ -54,6 +54,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improvements
 
+- [#776](https://github.com/alleslabs/celatone-frontend/pull/776) Add expand/collapse all to proposal messages in detail page
+- [#771](https://github.com/alleslabs/celatone-frontend/pull/771) Adjust proposal detail
```
Comment (songwongtp) [inline CHANGELOG.md:58]: extra changelog

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/proposal-details/components/proposal-overview/ProposalMessages.tsx [hunk 4]
```diff
@@ -1,39 +1,73 @@
...
+      <Accordion
+        allowMultiple
+        width="full"
+        variant="transparent"
+        index={expandedIndexes}
+        onChange={(indexes: number[]) => setExpandedIndexes(indexes)}
+      >
+        {messages?.map((item, i) => (
+          <ProposalMessageCard
+            key={`msg-${item}`}
```
Comment (evilpeach) [inline src/lib/pages/proposal-details/components/proposal-overview/ProposalMessages.tsx:65]:
```
  key={`msg-${item["@type"]}`}
  ```

File: src/lib/pages/proposal-details/components/proposal-overview/ProposalMessages.tsx [hunk 5]
```diff
@@ -1,39 +1,73 @@
...
+          rightIcon={
+            <CustomIcon
+              name={expandedIndexes.length ? "chevron-up" : "chevron-down"}
+              boxSize={3}
+            />
+          }
+          onClick={() => {
+            trackUseExpandAll(
+              expandedIndexes.length ? "collapse" : "expand",
+              "account detail resources Tab"
```
Comment (evilpeach) [inline src/lib/pages/proposal-details/components/proposal-overview/ProposalMessages.tsx:46]: account detail?

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**